src/routes/(auth)/login/+page.svelte

```diff
@@ -20,18 +20,19 @@
     onSubmit: async ({ email, password }) => {
       setTimeout(async () => {
         const basicAuth = createHeader(email, password);
-        const response = await fetch('/login', {
+        const request = await fetch('/login', {
           method: 'POST',
           headers: {
             Authorization: basicAuth
           }
         });
 
-        if (response.ok) {
+        if (request.ok) {
           notification.notifySuccess('Logged in successfully');
           window.location.pathname = '/';
         } else {
-          notification.notifyFailure('Failed to login, try again');
+          const response = await request.json();
+          notification.notifyFailure(response.message);
```

Comment:
Its not recommended to log internal errors. These are intended for developer debugging.

Reply:
What types of errors do I have to display?

Reply:
You could check on `error.code` and build a message for the user based on that!